### PR TITLE
Km events automation test

### PR DESCRIPTION
**Pull request by kiley-mitti**
Created a public google calendar that Daniel Nutter can edit. The home page pulls in these events and displays them. If not, it falls back to a generic list of general events and a link to the calendar. The events aren't all in yet, working with Daniel to ensure they're up to snuff but the code should work regardless.

Check it out! 
[Jira Ticket](https://rocketcom.atlassian.net/browse/ASTRO-4830)

### CODE REVIEW

File: src/components/home/community/newevents.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import { h } from 'project:utils/html.ts'
```
Comment (jonathantneal) [inline src/components/home/community/newevents.js:1]: **Requests**:
- May we rename this file to something that communicates what it does? Maybe something like `home-community-events.update.js`?
- May we import shared methods from the shared source file?
-

**Review by jonathantneal — CHANGES_REQUESTED** [review]
Thanks for this. This looks great! I’ve requested some changes, but they’re mostly code organization & de-dupe stuff. :)

File: src/components/home/community/newevents.js (hunk 1 above)
Comment (kiley-mitti) [reply src/components/home/community/newevents.js]: ok! I think I made the appropriate changes. :) I'll rerequest review

**Comment by kiley-mitti** [comment]
@jonathantneal Sure thing! I've removed it. It's less relevant since the server side render makes sure that we are mostly up to date. We will want to make sure that it runs a nightly rebuild so that the site can grab fresh info.

**Pull request merged**